docs/sources/datasources/loki/query-editor/index.md

```diff
@@ -171,6 +171,8 @@ The following options are the same for both **Builder** and **Code** mode:
 
 - **Line limit** -Defines the upper limit for the number of log lines returned by a query. The default is `1000`
 
+- **Direction** - Determines the search order. `Backward` is a backward search starting at the end of the time range. `Forward` is a forward search starting at the beginning of the time range. The default is `Backward`
```

Comment:
I think these should be bold because they are a UI element? https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#bold

```suggestion
- **Direction** - Determines the search order. **Backward** is a backward search starting at the end of the time range. **Forward** is a forward search starting at the beginning of the time range. The default is **Backward**
```